### PR TITLE
Drop support for npm v4

### PR DESCRIPTION
**Pull request by bertdeblock**
Closes https://github.com/ember-cli/ember-try/pull/961.

### CODE REVIEW

**Comment by bertdeblock** [comment]
@kategengler @mansona Should I just remove the use of `--no-shrinkwrap` in this PR as well?

**Comment by mansona** [comment]
@bertdeblock yes I would say so 👍

**Review by kategengler — APPROVED** [review]

File: lib/dependency-manager-adapters/npm.js [hunk 1]
```diff
@@ -21,8 +21,6 @@ module.exports = CoreObject.extend({
   configKey: 'npm',
   packageJSON: 'package.json',
   packageJSONBackupFileName: 'package.json.ember-try',
-  npmShrinkWrap: 'npm-shrinkwrap.json',
```
Comment (bertdeblock) [inline lib/dependency-manager-adapters/npm.js:24]: @kategengler Not 100% sure, but can we remove backing up `npm-shrinkwrap.json` as well?

Comment (kategengler) [reply lib/dependency-manager-adapters/npm.js:24]: Yeah I think so

**Review by kategengler — APPROVED** [review]

**Pull request merged**